test/lit.cfg

```diff
@@ -338,6 +344,11 @@ if run_vers.endswith('-simulator'):
 else:
     run_environment=''
 
+target_arch = run_cpu
+if run_os == 'openbsd' and run_cpu == 'amd64':
+   target_arch = run_cpu
```

Comment:
Redundant as already set above, unless you meant to put it after the following line.